x-pack/legacy/plugins/apm/mappings.json

```diff
@@ -41,5 +41,34 @@
         }
       }
     }
+  },
+  "apm-ui-indices": {
+    "properties": {
+      "apm_oss": {
+        "properties": {
+          "sourcemapIndices": {
+            "type": "text"
```

Comment:
`text` is mostly used for full-text searches. It will go through the analyzer, will be tokenized and such. 
Unless we want that we should use `keyword`